File: Python/palindromnumber_09.py
# https://leetcode.com/problems/palindrome-number/
# just convert the number into string then reverse it and check with a copy of it . if the copy same then return true or else return false
class Solution:
    def isPalindrome(self, x: int) -> bool:
        valueString = str(x)
        copyValue = valueString
        copyValue = copyValue[::-1]
        if copyValue==valueString:
            return True
        else:
            return False

value = -121
obj = Solution()
ans = obj.isPalindrome(value)
print(ans)

File: javaScript/palindromNumber_09.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    const original = x.toString();
    const reversed = original.split('').reverse().join('');
    return original === reversed;
};


let kk = 121
isPalindrome(kk)